### ISSUE TITLE
More training details

### ISSUE THREAD

**Issue by patrickpoirson** [argman/EAST#17]
Hi,

I am trying to exactly reproduce your released model. Could you provide some more details about the training. In the readme it looks like you use 14 images per gpu and I see you've mentioned training with 4 gpus? Was your total batch size then 56? Did you adjust the learning rate at all for such large batch size or was the default one used?

Also, you mention using icdar2013 training set as well. Anything special here or is sampling between icdar2015 and 2013 1:1. 

Any more details that you think may be relevant?

Btw. Small typo in the readme "Thanks for the author's (@zxytim) help! Please site his paper if you find this useful." site -> cite

Thanks for releasing the code. It's great!

**Comment by argman** [comment]
1. I havenot do much tuning of the learning rate. you may just use the default setting and see the result, we can discuss about the learning rate schedule.
2. no special sampling, i just merged them together. but seems you can achieve comparable result only with icdar2015 dataset.

**Comment by patrickpoirson** [comment]
Thanks!
